src/api/db/migrate/20210112091454_change_channel_disabled_default.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeChannelDisabledDefault < ActiveRecord::Migration[6.0]
+  # rubocop:disable Rails/ReversibleMigration
+  def change
+    change_column_default :channels, :disabled, false
```

Comment:
Could this be written like that?

```suggestion
    change_column_default :channels, :disabled, from: nil, to: false
```

Reply:
It definitely could be written like that to "make rubocop happy". However I don't see a scenario where we are going rollback from treating boolean as boolean and therefore having just two states - true or false - to something where we introduce a third state: `nil`.  I would prefer to disable this Cop and rely on `strong_migrations`